server/chat-plugins/scavengers.js

```diff
@@ -33,6 +34,11 @@ const HISTORY_PERIOD = 6; // months
 
 const ScavengerGames = require("./scavenger-games");
 
+let scavengersData = {recycledHunts: []};
+try {
+	scavengersData = JSON.parse(FS.readIfExistsSync(DATABASE_FILE));
+} catch (e) {}
```

Comment:
Shouldn't be in `try`/`catch` anymore, `readIfExistsSync` should handle this for you.

Reply:
`FS.readIfExistsSync` just returns a string and if it returns the empty string (what it does if the file doesn't exist) then should we catch the error thrown by `JSON.parse`?